src/lib/openapi/spec/feature-toggle-dto-schema.ts

```diff
@@ -0,0 +1,33 @@
+import { createSchemaObject, CreateSchemaType } from '../types';
+
+const schema = {
+    type: 'object',
+    required: ['name'],
+    properties: {
+        name: {
+            type: 'string',
+        },
+        description: {
+            type: 'string',
+        },
+        type: {
+            type: 'string',
+        },
+        stale: {
+            type: 'string',
+        },
+        archived: {
+            type: 'string',
+        },
+        createdAt: {
+            type: 'string',
+        },
+        impressionData: {
+            type: 'boolean',
+        },
+    },
+} as const;
+
+export type FeatureToggleDtoSchema = CreateSchemaType<typeof schema>;
+
+export const featureToggleDtoSchema = createSchemaObject(schema);
```

Comment:
Could call this `updateFeatureSchema`, to match `createFeatureSchema`. 

Not sure if they should also be the same schema, or if creates and updates have differing data.